templates/mod/mime.conf.erb

```diff
@@ -31,6 +31,6 @@ AddLanguage sv .sv
 AddLanguage zh-CN .zh-cn
 AddLanguage zh-TW .zh-tw
 
-AddHandler type-map var
-AddType text/html .shtml
-AddOutputFilter INCLUDES .shtml
+<%- @mime_types_additional.sort.each do |key, value_hash| -%>
+        <%- value_hash.each do |value_hash_key, value_hash_value| %><%=key%> <%=value_hash_key%> <%=value_hash_value%><% end %>
```

Comment:
hmmmm...it shows 3 lines in actual file...its strange that  the diff is showing it on one line?
do you mean to refactor the second line with extra indenting?
<%- @mime_types_additional.sort.each do |key, value_hash| -%>
        <%- value_hash.each do |value_hash_key, value_hash_value| %><%=key%> <%=value_hash_key%> <%=value_hash_value%><% end %>
<% end %>